flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/BangEqualITCase.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.runtime.stream.sql
+
+import org.apache.flink.streaming.api.scala._
+import org.apache.flink.table.api.bridge.scala._
+import org.apache.flink.table.data.RowData
+import org.apache.flink.table.planner.runtime.utils.{StreamingTestBase, TestingAppendRowDataSink}
+import org.apache.flink.table.runtime.typeutils.InternalTypeInfo
+import org.apache.flink.table.types.logical.{IntType, VarCharType}
+
+import org.junit.Assert._
+import org.junit.Test
+
+class BangEqualITCase extends StreamingTestBase {
+
+  @Test
+  def testBangEqual(): Unit = {
+
+    val sqlQueryBangEqual =
+      """
+        |SELECT * FROM 
+        | (VALUES (1, 'Bob'), (1, 'Alice'), (2, 'Lily')) T(a, b)
+        |WHERE a != 2
+        |
+        |""".stripMargin
+
+    val sqlQueryUnEqual =
+      """
+        |SELECT * FROM 
+        | (VALUES (1, 'Bob'), (1, 'Alice'), (2, 'Lily')) T(a, b)
+        |WHERE a <> 2
```

Comment:
Would the sql statement be compressed  by extracting '<>' & '!=' into a local variable for reusing code ? 
Or split the test case into two test cases, testBangEqual() & sqlQueryUnEqual().
(only a minor comment.)

Reply:
In fact, '<>'Already covered by another case, I just want to show that '!=' gives the same result, let's just keep '!= '.